web/ui/react-app/src/pages/targets/target.ts

```diff
@@ -0,0 +1,42 @@
+export interface Labels {
+  [key: string]: string;
+}
+
+export interface Target {
+  discoveredLabels: Labels;
+  labels: Labels;
+  scrapePool: string;
+  scrapeUrl: string;
+  lastError: string;
+  lastScrape: string;
+  lastScrapeDuration: number;
+  health: string;
+}
+
+export interface TargetGroupMetadata {
```

Comment:
This seems like premature abstraction, I'd just add an `upCount` field directly to the `TargetGroup`.

Reply:
done